core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerPool.scala

```diff
@@ -78,6 +76,7 @@ case class DeletionContainer(deletionMessage: ContainerDeletionMessage)
 case object Remove
 case class Keep(timeout: FiniteDuration)
 case class PrewarmContainer(maxConcurrent: Int)
+case object GetState
```

Comment:
Is there any reason we should define another `GetState` here?

Reply:
I see what happened now, I had this in progress for awhile and had a bad merge somewhere with your recent commits. Should all be cleaned up now.